docs/reference/stable-patch-format.rst

```diff
@@ -219,10 +204,10 @@ Comment body
      .. code-block:: none
 
         (cherry picked from commit <sha1> <upstream repo name>)
- 
-   .. note::
 
-      Omit the "<upstream repo name>" if the patch comes from the mainline tree.
+     .. note::
```

Comment:
hi @lkpdn , the indentation here seems to imply that this note only applies to cheery-picked patches.
Is that the case?

Also I think there was a question about this <upstream repo name> and what should be indicated here.
Maybe this note needs to be mentioned directly instead.

Reply:
For example

   Backported patches:
   - If the patch required changes (e.g. it did not apply cleanly), use
     "backported from commit <sha1> <upstream repo name>" between brackets "()". Omit the "<upstream repo name>" if the patch comes from the mainline tree. For example: